cli/src/commands/invite/claim.rs

```diff
@@ -89,6 +150,20 @@ async fn step0(
     Ok(ctx)
 }
 
+/// Step 0.5: choose recipient
+fn shamir_pick_recipient(
+    ctx: &ShamirRecoveryClaimPickRecipientCtx,
+) -> anyhow::Result<ShamirRecoveryClaimInitialCtx> {
+    let recipients = ctx.recipients_without_a_share();
+    let human_recipients: Vec<&_> = recipients.iter().map(|r| &r.human_handle).collect();
+    let selection = Select::new()
+        .default(0)
+        .with_prompt("Choose a person to contact now")
+        .items(&human_recipients)
+        .interact()?;
```

Comment:
Here we ask the user to pick the next recipient but we do not display the current progress (like `2 out of 3 shares already retrieved`). Also, we don't show how the number of shares owned by each recipient.